src/Elasticsearch/Framework/Indexing/ElasticsearchIndexer.php

```diff
@@ -149,8 +149,6 @@ private function init(array $entities = []): IndexerOffset
     {
         $this->connection->executeStatement('DELETE FROM elasticsearch_index_task');
 
-        $this->createScripts();
```

Comment:
The private is marked as deprecated but are not removed in 6.6, see MR in NEXT-29212